sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/cosmos/AzureCosmosProperties.java

```diff
@@ -32,6 +33,9 @@
     @NestedConfigurationProperty
     private final HttpProxyProperties proxy = new HttpProxyProperties();
 
+    @NestedConfigurationProperty
+    private final RetryProperties retry = new RetryProperties();
```

Comment:
How about putting this retry in the Service still, and override it in the HTTP?

Reply:
OK